config/environments/production.rb

```diff
@@ -19,7 +19,7 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # config.assets.js_compressor = :uglifier DOES NOT WORK ON HEROKU
```

Comment:
I mean, I think you are both including the uglifier gem and disabling uglifier from being run. I *think* it would work if you just do one or the other, unless there's something I'm missing.